tutorials/tmva/TMVAMulticlass.C

```diff
@@ -101,7 +108,9 @@ void TMVAMulticlass( TString myMethodList = "" )
       TString createDataMacro = gROOT->GetTutorialDir() + "/tmva/createData.C";
       gROOT->ProcessLine(TString::Format(".L %s",createDataMacro.Data()));
       gROOT->ProcessLine("create_MultipleBackground(2000)");
-      std::cout << " created tmva_example_multiple_background.root for tests of the multiclass features"<<std::endl;
+      // rename file to avoid clash con other tutorials that need onlmy 200 events
+      gSystem->Exec("mv tmva_example_multiple_background.root tmva_example_multiclass.root"); 
```

Comment:
This still breaks if two tests run concurrently and it just happens that they create the file at the same time, right?

Reply:
You can fetch the file directly from here via davix: http://root.cern.ch/files/tmva_multiclass_example.root
The file has also 2000 events!

Reply:
Thanks for the suggestion. I did not know the file was available there